ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/snapsync/request/heal/AccountFlatDatabaseHealingRangeRequest.java

```diff
@@ -95,12 +95,12 @@ public Stream<SnapDataRequest> getChildRequests(
       downloadState.getMetricsManager().notifyRangeProgress(HEAL_FLAT, endKeyHash, endKeyHash);
     }
 
-    Stream.of(existingAccounts.entrySet(), removedAccounts.entrySet())
+    Stream.of(existingAccounts.entrySet(), flatDbAccounts.entrySet())
         .flatMap(Collection::stream)
```

Comment:
non blocking, but Streams.concat() would be more readable than Stream.of(..).flatMap(Collection::stream).

Reply:
do you prefer something like that ? 

```java
Stream.concat(existingAccounts.entrySet().stream(), flatDbAccounts.entrySet().stream())
        .forEach(
        ```